mods/ra/maps/soviet-04a/reinforcements_teams.lua

```diff
@@ -0,0 +1,84 @@
+Civs = { civ1, civ2, civ3 }
+Village = { civ1, civ2, civ3, village1, village2, village3, village4, village5 }
```

Comment:
Could you remove `village3` and `village4` from this list?
IMHO they aren't needed as they don't get revealed by the camera.